PaolaSantolloV/react-apprenticeship-capstone1
# Mini challange1

## Description

This PR contain the Mini-Challenge 1: Core Concepts and Styling and Mini-Challenge 2: Intro to Testing

- The reusable components was created
- Add Styled-components and ReactIcons library to the project
- The test for some components and pages was created


## Type of Change



## How should this be tested?

In your terminal run `npm install` after that `npm run start`, now you can see the page on http://localhost:3000/
To run the test `npm run test`, if you want to watch the coverage `npm run test a -- --coverage`.

## Any background context you want to provide?

On the Login modal you can write wherever username and password, this form isn't validate yet.

## What is the deliverable?

First deliverable

## Screenshots and links

Link netfly:  https://capstone1project.netlify.app/
Coverage test: 77.61%

![Screen Shot 2022-01-12 at 12 39 10](https://user-images.githubusercontent.com/97254325/149202874-b94c3c9e-7f70-463e-89c1-54b807330aa7.png)


## Notes

This PR contain the mini-challange 1 and 2

## Checklist
